hw/ip/csrng/doc/_index.md

```diff
@@ -541,6 +541,12 @@ The conditions that cause this to happen are either when there is a push to a fu
 
 # Programmers Guide
 
+## Module enable and disable
+
+CSRNG may only be enabled if `ENTROPY_SRC` is enabled.
```

Comment:
For known-answer testing, it is possible to enable CSRNG without enabling ENTROPY_SRC. I've tested this for switching off the masking in AES. It works. However, I am not sure if we need to mention this option here. It probably just creates confusion.